pyproject.toml

```diff
@@ -48,8 +48,8 @@ dependencies = [
   "orjson==3.10.18",
   "prometheus-client==0.22.1",
   # "psycopg2-binary==2.9.10",
-  "psycopg==3.1.18",
-  "psycopg-binary==3.1.18",
+  "psycopg==3.2.10",
+  "psycopg-binary==3.2.10",
```

Comment:
Living on the edge curious if you guys can get it to work without the locking problem.